packages/graphql-dynamodb-transformer/src/DynamoDBModelTransformer.ts

```diff
@@ -105,6 +105,9 @@ export class DynamoDBModelTransformer extends Transformer {
         nonModelArray.forEach(
             (value: ObjectTypeDefinitionNode) => {
                 let nonModelObject = makeNonModelInputObject(value, nonModelArray, ctx)
+                if (this.typeExist(nonModelObject.name.value, ctx)) {
```

Comment:
Instead of the return why not just check for the negative. `if(!this.typeExists(...)) { ctx.addInput() }`